### DISCUSSION TITLE
Many 2 Many - is this a bug?

### DISCUSSION THREAD

**Discussion by TheOneValen** [nocodb/nocodb#4604] [Q&A] [answered]
After creating and populating a many to many relationship, i have 2 new tables

![grafik](https://user-images.githubusercontent.com/4579392/205636515-c4ff05e4-ef28-44ce-bc79-3ad43e081c70.png)

Is this a bug, a feature or a configuration issue? I did not find a bug but honestly do not know what exactly to search for.

I created 2 m2m relations between two tables. Version is 100.1

Rough structure of my project:

Table: Parts
Table: Projects

m2m relation Projects-Parts: "Needed Items"
m2m relation Projects-Parts: "Used Items"

**Comment by wingkwong** [comment, marked as answer]
Nope. It's expected to have such table. This is a M2M table created by NocoDB which is used to store the relationship (e.g. ID mapping) of both table. You can hide those M2M tables under Project Settings.

![image](https://user-images.githubusercontent.com/35857179/205645378-b89e2bfc-8cb8-4944-a604-ceb0a909d05f.png)

**Reply by o1lab** [reply]
More details here in [docs](https://docs.nocodb.com/setup-and-usages/meta-management#miscellaneous)

Also, @TheOneValen did you do delete some tables and did a [schema sync](https://docs.nocodb.com/setup-and-usages/sync-schema) ? which often leads to showin the old m2m tables.

**Reply by TheOneValen** [reply]
Yes I did modify and delete stuff, then sync. Using the settings under "Misc" they are hidden again, thank you.

I Do have a different result than in the docs though:

The table for the relation is not shown in my ERD view, also only one of the M2M relationships is shown, the other is not.
![grafik](https://user-images.githubusercontent.com/4579392/205656312-4c47546f-2b77-4305-9177-f111d53ff12d.png)

**Reply by o1lab** [reply]
Will need steps to reproduce the problem. Please create an issue.

**Reply by TheOneValen** [reply]
#4607
